source/jormungandr/tests/heat_maps_tests.py

```diff
@@ -114,10 +115,12 @@ def test_heat_maps_from_stop_point(self):
         response = self.query(q)
 
         is_valid_heat_maps(response, self.tester, q)
-        assert get_duration(stopB_coord, response) == 7  # about 0
-        assert get_duration(s_coord, response) == 20  # about 18
-        assert get_duration(stopA_coord, response) == 9  # about 2
-        assert get_duration(r_coord, response) == 74  # about 2 + 81
+
+        tolerance = 10
```

Comment:
As discussed (for future questions): this tolerance is added because we currently can't pin the grid in heatmap, so it moves when adding data... this should protect a bit next data change.